WORKSPACE

```diff
@@ -39,6 +46,20 @@ http_archive(
     urls = ["https://github.com/google/benchmark/archive/master.zip"],
 )
 
+# Used by gRPC.
+http_archive(
```

Comment:
You shouldn't need this particular `http_archive` invocation unless for some reason you really want `master` branch (which, btw, makes your build non-hermetic). You're already getting an instance from your call to `grpc_deps()`. It's just the `load()` and `apple_rules_dependencies()` invocations below that are necessary due to https://github.com/bazelbuild/bazel/issues/1550.

Reply:
Thanks! How does https://github.com/census-instrumentation/opencensus-cpp/pull/379 look to you?

As for master, we are trying to follow Abseil's "live at head" philosophy. (...on our own master branch; whereas OpenCensus releases pin their deps)